src/context/CapturesContext.js

```diff
@@ -69,44 +60,49 @@ export function CapturesProvider(props) {
     });
   };
 
-  const getCaptureCount = async () => {
-    log.debug('load capture count');
-    const paramString = `where=${JSON.stringify(filter.getWhereObj())}`;
-    const response = await queryCapturesApi({
-      count: true,
-      paramString,
-    });
-    const { count } = response.data;
-    setCaptureCount(Number(count));
-  };
-
-  const getCapturesAsync = async () => {
+  const getCaptures = async () => {
     log.debug('4 - load captures');
+
     const filterData = {
-      where: filter.getWhereObj(),
-      order: [`${orderBy} ${order}`],
+      ...filter.getWhereObj(),
+      order_by: orderBy,
+      order,
       limit: rowsPerPage,
-      skip: page * rowsPerPage,
+      offset: page * rowsPerPage,
     };
-    const paramString = `filter=${JSON.stringify(filterData)}`;
+
     setIsLoading(true);
-    const response = await queryCapturesApi({ paramString });
+    const response = await queryCapturesApi(filterData);
     setIsLoading(false);
-    setCaptures(response.data);
+    setCaptures(response?.data?.captures);
+    setCaptureCount(Number(response?.data?.total));
   };
 
   // GET CAPTURES FOR EXPORT
   const getAllCaptures = async () => {
     log.debug('load all captures for export');
     const filterData = {
-      where: filter.getWhereObj(),
+      ...filter.getWhereObj(),
       order: [`${orderBy} ${order}`],
       limit: 20000,
     };
+    const { data } = await queryCapturesApi(filterData);
+    return data;
+  };
+
+  const getCapture = (id) => {
+    setIsLoading(true);
 
-    const paramString = `filter=${JSON.stringify(filterData)}`;
-    const response = await queryCapturesApi({ paramString });
-    return response;
+    api
+      .getCaptureById(`${process.env.REACT_APP_QUERY_API_ROOT}/v2/captures`, id)
+      .then((res) => {
+        setIsLoading(false);
+        setCapture(res);
+      })
+      .catch((err) => {
+        setIsLoading(false);
+        console.error(`ERROR: FAILED TO GET SELECTED TREE ${err}`);
```

Comment:
Use `loglevel` rather than `console.error`, and it's a capture rather than a tree 😄